### ISSUE TITLE
.net core netstandard1.6

### ISSUE THREAD

**Issue by NickolasAcosta** [castleproject/Core#191]
I'm currently upgrading a library from .net core RC1 to .net core rtm.

When using the new version of moq I get the error:
The dependency Castle.Core 4.0.0-alpha001 does not support framework .NETStandard,Version-v1.6

I didn't see an issue when I searched so I decided to open one.
https://github.com/moq/moq4/issues/168

**Comment by jeremymeng** [comment]
@NickolasAcosta The current release package doesn't support netstandard.  Would adding an imports of dotnet5.4 work around it for you?

```javascript
    "netstandard1.6": {
      "imports": [ "dotnet5.4" ]
    }
```

**Comment by NickolasAcosta** [comment]
Temporarily I guess, I think the goal is to target as much as possible.  Thank you.

**Comment by jonorossi** [comment]
We are supporting .NET Standard 1.4+, not 1.6+.

The problem with `Unsupported0.0` was fixed a couple of months ago in #174, but a new package was never released. The plan is to get a new package out once we get a green build.
